apis/v1alpha1/httproute_types.go

```diff
@@ -432,19 +432,39 @@ const (
 // HTTPRequestHeaderFilter defines configuration for the RequestHeaderModifier
 // filter.
 type HTTPRequestHeaderFilter struct {
-	// Add adds the given header (name, value) to the request
+	// Set overwrites the request with the given header (name, value)
 	// before the action.
 	//
 	// Input:
 	//   GET /foo HTTP/1.1
+	//   my-header: foo
 	//
 	// Config:
-	//   add: {"my-header": "foo"}
+	//   set: {"my-header": "bar"}
 	//
 	// Output:
 	//   GET /foo HTTP/1.1
+	//   my-header: bar
+	//
+	// Support: Extended
+	// +optional
+	Set map[string]string `json:"set,omitempty"`
+
+	// Add adds the given header (name, value) to the request
+	// before the action. It appends to any existing values associated
+	// with the header name.
+	//
+	// Input:
+	//   GET /foo HTTP/1.1
 	//   my-header: foo
 	//
+	// Config:
+	//   add: {"my-header": "bar"}
+	//
+	// Output:
+	//   GET /foo HTTP/1.1
+	//   my-header: foo bar
```

Comment:
This seems incorrect. Header values are allowed to contain whitespaces.
The usual textual representation for multiple headers is:

```
	//   GET /foo HTTP/1.1
	//   my-header: foo
	//   my-header: bar
```

Reply:
Thank you! Updated.